_accounts.md

```diff
@@ -130,36 +130,36 @@ The following is a list of valid card data that can be used to add a card accoun
 
 Number           | Brand      | Type   | Country | OCT settlement |
 ---------------- | ---------- | ------ | ------- | -------------- |
-5502514549870410 | mastercard | debit  | FR      | standard       |
-5355223761921186 | mastercard | debit  | GB      | instant        |
-5573606426146833 | mastercard | debit  | GB      | instant        |
-5318773012490080 | mastercard | debit  | US      | instant        |
-4921817844445119 | visa       | debit  | GB      | instant        |
-4659105569051157 | visa       | debit  | GB      | instant        |
+4024007181869214 | visa       | credit | US      | N/A            |
```

Comment:
this one simulates `card_declined_by_bank` error, cannot be used to add a card account

Reply:
let's remove then.